ckanext/stcndm/logic/__init__.py

```diff
@@ -0,0 +1,702 @@
+__author__ = 'Statistics Canada'
+
+import functools
+import logging
+import re
+
+import formencode.validators
+
+import ckan.model as model
+import ckan.new_authz as new_authz
+import ckan.lib.navl.dictization_functions as df
+import ckan.plugins as p
+
+from ckan.common import _, c
+
+log = logging.getLogger(__name__)
+_validate = df.validate
+
+
+class AttributeDict(dict):
```

Comment:
Oh, I see this whole file was just copied from ckan.  I will remove this from ckan core.